packages/popup-component/src/components/popup/shared.scss

```diff
@@ -2,6 +2,7 @@ $blue: #d8f6ff;
 $red: #ff4630;
```

Comment:
After seeing this my comment above may need to use color variables

Reply:
Totally my bad, I forgot to place that into a variable